extensions/adobe/experience/adcloud/adconversions.schema.json

```diff
@@ -0,0 +1,225 @@
+  "definitions": {
+    "adconversions": {
+      "properties": {
+        "xdm:logId": {
+          "title": "Internal identifier for the batch",
+          "type": "number",
+          "description": "Internal identifier for the batch represented in the YYYYMMDDHH24 format"
+        },
+        "xdm:conversionTime": {
+          "title": "Conversion Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp that indicates when the conversion event occured"
+        },
+        "xdm:triggerTime": {
+          "title": "Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Timestamp that indicates when the click event occured"
+        },
+        "xdm:advertiserId": {
+          "title": "Advertiser Identifier",
+          "type": "string",
+          "description": "Advertiser ID for the campaign that initiated the conversion event"
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name"
+        },
+        "xdm:propertyName": {
+          "title": "Property Name",
+          "type": "string",
+          "description": "Property Name Configured by Pixel, Analytics, Custom Feeds"
+        },
+        "xdm:clientId1": {
+          "title": "Client ID1",
+          "type": "string",
+          "description": "Alphanumeric strings that are matched against clicks to extract keyword,creative,adgroup IDs along with keyword text and matchtype"
+        },
+        "xdm:clientId2": {
+          "title": "Client ID2",
+          "type": "string",
+          "description": "Alphanumeric strings that are matched against clicks to extract keyword,creative,adgroup IDs along with keyword text and matchtype"
+        },
+        "xdm:accountId": {
+          "title": "Account Identifier",
+          "type": "string",
+          "description": "Account ID for the search advertising platform campaign that initiated the conversion event"
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the advertising platform"
+        },
+        "xdm:placementId": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "description": "Placement ID on the display advertising platform. It will be populated only for non-search advertising related campaigns"
+        },
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform"
+        },
+        "xdm:adId": {
+          "title": "Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the advertising platform"
+        },
+        "xdm:term": {
```

Comment:
terms seems very genetic. I know you have the `Search` in the title, but since `term` is the field you will be using in the code, please use `searchTerm`.

Reply:
@skannan-adobe , This is neither a conversion search Term. This is basically a targeted keyword that matched the search Term. This is typically referred as a keywords in other platforms. I would stick to word term as it has already been used in other schemas

https://support.google.com/google-ads/answer/6323?hl=en